prog/vm/github_runner.rb

```diff
@@ -54,7 +54,8 @@ def pick_vm
       enable_ip4: true,
       arch: label_data["arch"],
       allow_only_ssh: true,
-      swap_size_bytes: 4294963200 # ~4096MB, the same value with GitHub hosted runners
+      swap_size_bytes: 4294963200, # ~4096MB, the same value with GitHub hosted runners
+      gpu_enabled: label_data["boot_image"].include?("-gpu-")
```

Comment:
You might add `enable_gpu: true` flag to these lines on the yaml. Then you can do `label_data["enable_gpu"] || false`

Reply:
I didn't want to add a new column, but sure.

Reply:
I think you can add the new column to only GPU ones, but it's up to you.